genfragments/PbPb_5p36TeV/Powheg/POWHEG_DYToEE_M_10_50-fragment.py

```diff
@@ -0,0 +1,47 @@
+import FWCore.ParameterSet.Config as cms
+
+externalLHEProducer = cms.EDProducer('ExternalLHEProducer',
+    args = cms.vstring('/cvmfs/cms.cern.ch/phys_generator/gridpacks/RunIII/5p36TeV/Powheg/el8_amd64_gcc11/Z_el8_amd64_gcc11_CMSSW_13_0_18_HeavyIon_Run3_PbPb_5p36TeV_DYToEE_M_10_50.tgz'),
+    nEvents = cms.untracked.uint32(5000),
+    numberOfParameters = cms.uint32(1),
+    outputFile = cms.string('cmsgrid_final.lhe'),
+    scriptName = cms.FileInPath('GeneratorInterface/LHEInterface/data/run_generic_tarball_cvmfs.sh'),
+    generateConcurrently = cms.untracked.bool(True)
+)
+
+#Link to datacards:
+#https://github.com/cms-sw/genproductions/blob/master/bin/Powheg/production/Run3/PbPb_5p36TeV/DYToLL/DYToEE_EPPS21_MllBinned_M_10_50.input
+
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.MCTunesRun3ECM13p6TeV.PythiaCP5Settings_cfi import *
+from Configuration.Generator.Pythia8PowhegEmissionVetoSettings_cfi import *
+from Configuration.Generator.PSweightsPythia.PythiaPSweightsSettings_cfi import *
+
+generator = cms.EDFilter("Pythia8ConcurrentHadronizerFilter",
+    PythiaParameters = cms.PSet(
+        pythia8CommonSettingsBlock,
+        pythia8CP5SettingsBlock,
+        pythia8PowhegEmissionVetoSettingsBlock,
+        pythia8PSweightsSettingsBlock,
+        processParameters = cms.vstring(
+            'POWHEG:nFinal = 1', ## Number of final state particles
+            ## (BEFORE THE DECAYS) in the LHE
+            ## other than emitted extra parton
+            'TimeShower:mMaxGamma = 4.0',
+            'BeamRemnants:primordialKThard=2.13'#kT for 5.36 TeV from GEN-22-001 slope
```

Comment:
the intrinsic kt is only valid for MG FXFX, please remove it from here. e.g., DY powheg-minnlo sample has very different kt value at 13TeV

Reply:
Done in 47aa9094132b9416acbf06d44ce90a48f6c5d823